shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.presentation.model
+
+data class SurveyUI (
```

Comment:
On Android, we have `SurveyUiModel`, so you can follow it

Reply:
@suho and maybe we can apply that for iOS also, just a thought 😆

Reply:
Renamed in https://github.com/blyscuit/kmm-swiftUI-IC/pull/57/commits/e9922606fe89a34ab57dcee3942dd7ded99ca3a6